validator/project_validator.go

```diff
@@ -1550,7 +1551,7 @@ func validateGenerateTasks(p *model.Project) ValidationErrors {
 
 // validateTaskSyncSettings checks that task sync in the project settings have
 // enabled task sync for the config.
-func validateTaskSyncSettings(p *model.Project, ref *model.ProjectRef) ValidationErrors {
+func validateTaskSyncSettings(p *model.Project, ref *model.ProjectRef, config *model.ProjectConfig) ValidationErrors {
```

Comment:
Since validateTaskSync doesn't use config, can write this as _ *model.ProjectConfig to indicate that it's not used.